scrapi/processing/elastic_search.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from elasticsearch import Elasticsearch
+
+from scrapi import settings
+from scrapi.processing.base import BaseProcessor
+
+es = Elasticsearch(
+    settings.ELASTIC_URI,
```

Comment:
The class is constantly torn down and built back up, but the module is only imported once. It's relatively expensive to make a new connection, so I figured it would be better to keep it outside (just like the cassandra processor).